src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_helpers.cc

```diff
@@ -22,4 +28,45 @@ RCAppExtensionPtr RCHelpers::GetRCExtension(
           extension_interface);
   return extension;
 }
+
+smart_objects::SmartObjectSPtr RCHelpers::CreateUnsubscribeRequestToHMI(
+    const std::string& module_type, const uint32_t correlation_id) {
+  using namespace smart_objects;
+  namespace commands = application_manager::commands;
+  namespace am_strings = application_manager::strings;
+
+  SmartObjectSPtr message = utils::MakeShared<SmartObject>(SmartType_Map);
+  SmartObject& ref = *message;
```

Comment:
@LuxoftAKutsan `ref` name is absolutely meaningless

Reply:
@AByzhynar fixed